packages/react-component-library/cypress/support/consoleErrors.ts

```diff
@@ -0,0 +1,19 @@
+import { Cypress, cy } from 'local-cypress'
+
+function check() {
+  let stub
+
+  Cypress.on('window:before:load', (win) => {
+    stub = cy.stub(win.console, 'error')
+  })
+
+  Cypress.on('command:end', () => {
+    if (stub) {
+      chai.expect(stub, 'There are console errors').to.have.callCount(0)
```

Comment:
I wonder if a message like 'There should be no console errors' would be clearer in the Cypress log?

![image](https://user-images.githubusercontent.com/66470099/139664352-4624e7f1-9099-47f9-8c62-1ba3ee10ee57.png)